src/NHibernate.Test/Async/NHSpecificTest/NH1553/MsSQL/SnapshotIsolationUpdateConflictTest.cs

```diff
@@ -77,16 +77,17 @@ private void SavePerson(Person p)
 		/// Tests, that NHibernate detects the update conflict and returns a StaleObjectStateException as expected.
 		/// This behaviour is part of the standard NHibernate feature set.
 		/// </summary>
+		/// <param name="cancellationToken">A cancellation token that can be used to cancel the work</param>
 		[Test]
-		public async Task UpdateConflictDetectedByNHAsync()
+		public async Task UpdateConflictDetectedByNHAsync(CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
@maca88 that's strange

Reply:
Sorry, I overlooked this change. I've released version `0.8.2.3` which doesn't generate it.

Reply:
Thanks again @maca88, but these changes are still there with 0.8.2.3.

Reply:
I tried to regenerate the code twice before releasing and the change was never there. Then I remembered that this issue occurred randomly depending on the order of the methods being analyzed, which was discovered and fixed when [#86](https://github.com/maca88/AsyncGenerator/issues/86) was done. I've ported [the fix](https://github.com/maca88/AsyncGenerator/commit/9e9b450bba341c30aa7384818475a2e992d9941a) and released it in `0.8.2.4`. Again, sorry for the confusion.

Reply:
No trouble, do not be sorry, it is already great you take the time to back-port things to old versions.